app/editor/src/features/admin/topics/TopicList.tsx

```diff
@@ -39,6 +39,20 @@ export const TopicList: React.FC = () => {
 
   const topicTypeOptions = getEnumStringOptions(TopicTypeName);
 
+  React.useEffect(() => {
```

Comment:
We still have the same implementation issue.  Why are we manually editing the DOM inline styling attributes?  Just change the CSS in the styled component.

Reply:
Updated with a different approach here, and it seems changing the CSS in the styled parent component won't work as that will affect the other parts of the system. Thanks

Reply:
If we have to do some logic to determine the overflow, can we send a prop to the styled component instead? @CelloLiu

Reply:
As the 'body' element is outside the React component tree, we might not be able to send a prop down to the styled component. However, please never mind, I have removed it now as this is only for fixing an edge case. Thanks